cmd/rerunfails.go

```diff
@@ -51,23 +62,37 @@ func rerunFailsFilter(o *options) testCaseFilter {
 	return testjson.FilterFailedUnique
 }
 
+// Need to add the cov file context and etc...
```

Comment:
Is this a TODO that still needs to be resolved?

Reply:
No. I will remove.